plugins/inputs/beat/beat.go

```diff
@@ -0,0 +1,278 @@
+package beat
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	jsonparser "github.com/influxdata/telegraf/plugins/parsers/json"
+)
+
+const sampleConfig = `
+  ## An URL from which to read Beat-formatted JSON
+  ## Default is "http://127.0.0.1:5066".
+  url = "http://127.0.0.1:5066"
```

Comment:
Would it make sense to collect from multiple hosts? Then this should be a list...

Reply:
Think this would be of limited use, I imagine the most common deployments would be a local Telegraf agent and Beat agent per host. So I'd rather address this in a future PR if there's a need that isn't satisfied by a separate configuration per host.

Reply:
Fair enough, please make as resolved.